web/src/index.scss

```
@@ -75,14 +167,11 @@ body {
-/* Light mode gradient (default) */
+/* Flat surface background (seerbeta) */
 body {
-  background: linear-gradient(247.18deg, #f4f0fd 0%, #fdfff7 100%);
-}
-
-/* Dark mode gradient */
-:root[data-theme="dark"] body {
-  background: linear-gradient(247.18deg, #0b0e11 0%, #1a1f26 100%);
+  background: var(--bg);
+  color: var(--ink);
+  font-family: "Inter", system-ui, -apple-system, "Helvetica Neue", sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)